src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/AbstractEntityAIGuard.java

```diff
@@ -101,6 +102,55 @@
      */
     private BlockPos currentPatrolPoint = null;
 
+    /**
+     * The value of the speed which the guard will move.
+     */
+    private static final double ATTACK_SPEED = 0.8;
+
+    /**
+     * Indicates if strafing should be moving backwards or not.
+     */
+    private boolean strafingBackwards = false;
```

Comment:
### Errors:
- Field initialization to false is redundant 

### Warnings:
- Instance field name strafingBackwards doesn't match regex 'm_[a-z][A-Za-z\d]*'